Doc/library/asyncio-task.rst

```diff
@@ -279,8 +279,10 @@ Sleeping
    ``sleep()`` always suspends the current task, allowing other tasks
    to run.
 
-   The *loop* argument is deprecated and scheduled for removal
-   in Python 3.10.
+   .. deprecated-removed:: 3.8 3.10
+
+      The *loop* argument is deprecated and scheduled for removal
```

Comment:
The text should be updated.
Now it looks like:

*Deprecated since version 3.8, will be removed in version 3.10:*  The *loop* argument is deprecated and scheduled for removal in Python 3.10

Looks like battology.

Reply:
I spotted that and didn't wanted to change too much at once, but sure I can take care of it.